docs/advanced_usage/9_parallelism.md

```diff
@@ -1,41 +1,76 @@
 # Parallelism
 
-SMAC supports multiple workers natively via Dask. Just specify ``n_workers`` in the scenario and you are ready to go. 
-
+To facilitate fast execution, SMAC supports executing multiple workers simultaneously via [Dask](https://www.dask.org/). Using this functionality, splits SMAC into a main, and executor jobs.
+The main job handles the optimization process, and coordinates the executor jobs. The executors are queried with the target function and hyperparameter configurations, execute them, and return their result. 
```

Comment:
is the main job also run on a dask worker? I have the feeling, not. Rather main process + dask workers which are open and which get job packages